### ISSUE TITLE
Words with 'w' in the middle add extra spaces

### ISSUE THREAD

**Issue by geoffwild11** [BlueHatbRit/mdpdf#38]
I'm running into an issue where converting a file to PDF is adding spaces next to all of my lowercase w's in the middle of words. It seems to generally be only within the main text. It just looks pretty bad when I want to share this with others at work. I'm running this on a Windows 10 machine.

Also, I'm using [Markdown-to-PDF](https://github.com/travs/markdown-pdf) in [Atom](https://atom.io/). There is already a [bug ](https://github.com/travs/markdown-pdf/issues/178) reported there for this, but it looks like the problem is in the conversion.

[test.pdf](https://github.com/BlueHatbRit/mdpdf/files/1992388/test.pdf)

What it should look like:

# here is my file with some w's in places

What? where? how? how now brown cow?

wa wb wc wd we wf wg wh wi wj wk wl wm wn wo wp
wq wr ws wt wu wv wx wy wz w!!!w!

---
Here's the text inside it (added monospacing to preserve all characters)
```
# here is my file with some w's in places

What? where? how? how now brown cow?

wa wb wc wd we wf wg wh wi wj wk wl wm wn wo wp
wq wr ws wt wu wv wx wy wz w!!!w!
```

**Comment by BlueHatbRit** [comment]
Hey @geoffwild11, thanks for reporting this, you're right that this is most probably in mdpdf. mdpdf uses a browser to render the text so some of this will depend on the font, and some of it on the browser. It seems this is an issue with the lowercase `w` that's being used (thanks for the pdf).

The good news is, I'm actually starting to move mdpdf from phantom over to Chrome Puppeteer as it now finally supports all the pdf settings we need (literally just in the past few weeks). This should actually fix a lot of rendering issues and make the pdf's output as if done by Chrome manually.

#39 is the issue tracking this migration, if anyone wants to get their hands dirty then that would be fantastic as my time is quite limited at the moment.

**Comment by BlueHatbRit** [comment]
This _should_ be solved in 2.0.0, but it's worth checking the release notes before upgrading!
